public/theme/extensions-2021/js/vendor/image-scroll.js

```diff
@@ -0,0 +1,238 @@
+/**
```

Comment:
As far as I know, this is only used on the showcases page. Let's remove it.

Reply:
No, deleted.